source/server/index.js

```diff
@@ -1,3 +1,8 @@
-import ServerApp from '/source/imports/application/server-application.js'
+import ServerApp from '../imports/application/server-application.js';
 
-ServerApp.start();
\ No newline at end of file
+Meteor.startup(function() {
+
+  if (Meteor.isServer) {
+    ServerApp.start();
```

Comment:
If I do this than I will not be able to import client and server application server instances and use for example `ClientApp.send()`. I would need to export instances of applications in `index.js` files which does not sound right to me. Thoughts?

https://github.com/meteor-space/projection-rebuilding-example/blob/70471a1afbd1c7c546f057714ee94d69f9a161c0/source/imports/ui/actions/bank-accounts.js#l28